### PR TITLE
Bootstrapping topic

### PR DESCRIPTION
**Pull request by jimfulton**
Note that there are additions to the reference section solely to satisfy links from the new bootstrapping topic.

### CODE REVIEW

File: doc/topics/bootstrapping.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+Avoid modifying the python environment.
+  From a philosophical point of view, Buildout has tried to be
+  isolated from its environment, and requiring the Python environment
+  to be modified, by installing Buildout, was inconsistent.
+
+  Before `virtualenv <https://virtualenv.pypa.io/en/stable/>`_
+  existed, it might not have been possible to modify the environment
+  without building Python from source.
+
+  Unfortunately, doing this this requires :ref:`using a bootstrap script
```
Comment (leorochael) [inline doc/topics/bootstrapping.rst]: > ... doing this this requires...

Duplicated "this"

Comment (jimfulton) [inline doc/topics/bootstrapping.rst]: Fixed

**Comment by jimfulton** [comment]
Thanks for the speedy review.

**Comment by jimfulton** [comment]
I'll probably add some tests before I merge this.

**Pull request merged**